Lib/test/test_posix.py

```diff
@@ -231,6 +231,9 @@ def test_register_at_fork(self):
         with self.assertRaises(TypeError, msg="Invalid arg was allowed"):
             # Ensure a combination of valid and invalid is an error.
             os.register_at_fork(before=None, after_in_parent=lambda: 3)
+        with self.assertRaises(TypeError, msg="At least one argument is required."):
```

Comment:
```suggestion
        with self.assertRaises(TypeError, msg="At least one argument is required"):
```

Reply:
Follow other convention

Reply:
```
>>> os.register_at_fork()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: At least one argument is required.
```
Actual error does include `.` in the end, wouldn't test case fail then?
Also,wouldn't better approach be to remove `.` in `PyErr_SetString(PyExc_TypeError, "At least one argument is required.")` in Modules/posixmodule.c as well?